src/gui/widgets/CPULoadWidget.cpp

```diff
@@ -91,10 +91,21 @@ void CPULoadWidget::paintEvent( QPaintEvent *  )
 
 void CPULoadWidget::updateCpuLoad()
 {
-	// smooth load-values a bit
-	int new_load = ( m_currentLoad + Engine::audioEngine()->cpuLoad() ) / 2;
-	if( new_load != m_currentLoad )
+	// Additional display smoothing for the main load-value. Stronger averaging
+	// cannot be used directly in the profiler: cpuLoad() must react fast enough
+	// to be useful as overload indicator in AudioEngine::criticalXRuns().
+	const int new_load = (m_currentLoad + Engine::audioEngine()->cpuLoad()) / 2;
+
+	if (new_load != m_currentLoad)
```

Comment:
This won't update the tooltip unless the overall CPU load changes, even if the detailed loads do change. This is probably okay, but I wanted to check whether this was intended (and see what others think too).

Reply:
It's not quite intentional, but probably not a big issue either, because currently Qt refuses to update the tooltip unless the mouse cursor moves. So combined with the fact that the numbers jump all around the place even with relatively high averaging, it's only good for rough estimates anyway.